src/lib/components/autocompleteAddress/index.test.tsx

```diff
@@ -55,37 +55,41 @@ describe('AutocompleteAddress component', () => {
       }
     };
 
-    const screen = setup();
-    fireEvent.change(screen.getByTestId(inputTestId), {
+    const { getAllByTestId, getByTestId } = setup();
+
+    fireEvent.change(getByTestId(inputTestId), {
       target: { value: 'Köpeniker' },
     });
 
-    const inputs = screen.getAllByTestId(inputTestId);
-    expect(inputs[0].getAttribute('value')).toBe('Köpeniker Strasse');
+    const inputs = getAllByTestId(inputTestId);
+
     expect(inputs.length).toEqual(5);
+    expect(inputs[0].getAttribute('value')).toBe("Köpeniker Strasse");
```

Comment:
I would prefer to check if the text is visible on the screen. I.e. `screen.getByText('Köpeniker Strasse')`
What do you think?

Reply:
Makes sense, I'd suggest using the `getDisplayValue` instead:
`expect(getByDisplayValue("Köpeniker Strasse")).toBeVisible();`